### PR TITLE
Make role ready for production

### PR DESCRIPTION
**Pull request by Normo**
* use container images from [HIFIS fork](https://github.com/hifis-net/RSD-as-a-service) by default
* update variables and README.md
* remove symlink for env file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -91,7 +88,7 @@ Currently [supported platforms](meta/main.yml) are:
   - Description: GitHub personal access token
 - `rsd_jwt_secret`
   - Default: `changeme`
```
Comment (tobiashuste) [inline README.md:90]: Shortly after submitting my review I found this and thought, we could also add a 32 character default variable. This way people might read the description more carefully.


```suggestion
  - Default: `changemeChangemeChangemeChangeme`
```

**Review by tobiashuste — CHANGES_REQUESTED** [review]
Sorry, one comment ;)

**Review by christianhueserhzdr — DISMISSED** [review]
Dear Norman @Normo, I was not able to find any errors, inconsistencies or things to improve. All looks very good. I'll approve this PR.

File: README.md (hunk 1 above)
Comment (Normo) [reply README.md]: I updated it with 2112320683d63366be05b5dfc2243d58128ccb7c

**Review by tobiashuste — APPROVED** [review]
Thanks, @Normo :)

**Pull request merged**